cmake/onnxruntime_mlas.cmake

```diff
@@ -191,17 +211,19 @@ function(setup_mlas_source_for_windows)
     )
     set_source_files_properties(${mlas_platform_srcs_avx2} PROPERTIES COMPILE_FLAGS "/arch:AVX2")
 
+    set(mlas_platform_srcs_sse41
+      ${MLAS_SRC_DIR}/qgemm_kernel_sse41.cpp
+    )
+    set_source_files_properties(${mlas_platform_srcs_sse41} PROPERTIES COMPILE_FLAGS "-msse4.1")
```

Comment:
`-msse4.1` won't work for MSVC, right? that is the compiler for which `setup_mlas_source_for_windows()` will be called.